src/GrpcProtos/capture.proto

```diff
@@ -599,6 +599,28 @@ message ThreadStateSlice {
   // Otherwise, they should be 0 to minimize bandwidth usage.
   uint32 wakeup_tid = 7;
   uint32 wakeup_pid = 8;
+
+  // This is the callstack of the tracepoint that changed the state
+  // of the thread and caused this slice to happen. If the thread
```

Comment:
Does the callstack correspond to the beginning of the slice, or to the end of the slice? I assume the beginning? But it's not very clear.

Reply:
Added a sentence.